config/vufind/EDS.ini

```diff
@@ -187,18 +187,17 @@ top_rows = 2
 ; Should we OR together facets rather than ANDing them? Set to * for
 ; all facets, use a comma-separated list to apply to some of the facets, set
 ; to false or omit to disable ORed facets.
-; NOTE: In EDS, facet behavior is controlled through the EBSCO admin panel,
-; but you should configure this setting to match the EBSCO settings so that
-; VuFind displays match actual behavior.
+; Note that ContentProvider and SourceType are forced to be OR facets, ignoring
+; this setting, due to special handling by the EDS API.
+; Also note that there is incorrect behavior on any OR facet (other than the
+; forced OR facets) that already has one or more values applied to it, due to the
+; EDS API believing incorrectly that any additional applied values will combined
```

Comment:
Should this be "will be combined" rather than "will combined"?

Reply:
Thanks, fixed.